src/Application.hpp

```diff
@@ -60,6 +61,7 @@ class Application
     TaggedUsersController *const taggedUsers{};
     ModerationActions *const moderationActions{};
     TwitchServer *const twitch2{};
+    ChatterinoBadges *const badges{};
```

Comment:
This class should not be called "badges" - it's too generic. chatterinoBadges or just badges in a named struct `chatterino`, not sure which is the standard atm